cmd/gokr-packer/certs.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"github.com/gokrazy/internal/config"
```

Comment:
nit: for consistency, can you please move all github imports to a separate import group (separated by a blank line)?